src/figuresClasses.ts

```diff
@@ -2,10 +1,81 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+    public shape: 'triangle',
+  ) {
+    this.shape = 'triangle';
```

Comment:
Manually assigning the 'shape' property is unnecessary when using TypeScript's access modifiers. You can remove this line.